README.md

```diff
@@ -8,7 +8,8 @@ Your challenge is to build a simple and intuitive interface for searching throug
 
 ### Figma Design
 
-Here’s the design you’ll be implementing: **[Figma link](https://www.figma.com/design/AwC58Mqda2ZOjqzadPdiso/LES---Front-end-homework?node-id=0-1)**
+Here’s the design you’ll be implementing: **[Figma prototype](https://www.figma.com/proto/AwC58Mqda2ZOjqzadPdiso/LES---Front-end-homework?page-id=&node-id=1-4741&p=f&t=Dx21H0F2ytviGHvl-0&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=1%3A4741)**
+All the available material can be found from this link: **[Figma components](https://www.figma.com/design/AwC58Mqda2ZOjqzadPdiso/LES---Front-end-homework?node-id=0-1)**
```

Comment:
GitHub markdown needs two `\n` in order to add new line (which is why I suppose you were targetting).

Here is how it looks right now:

![1744382709](https://github.com/user-attachments/assets/f880fd4c-6a9b-423f-baf7-9c797a51f403)

I suggest:

```suggestion
Here’s the design you’ll be implementing: **[Figma prototype](https://www.figma.com/proto/AwC58Mqda2ZOjqzadPdiso/LES---Front-end-homework?page-id=&node-id=1-4741&p=f&t=Dx21H0F2ytviGHvl-0&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=1%3A4741)**.

All the available material can be found from this link: **[Figma components](https://www.figma.com/design/AwC58Mqda2ZOjqzadPdiso/LES---Front-end-homework?node-id=0-1)**.
```